src/i18n/locales/pt/translations.json

```diff
@@ -83,6 +83,10 @@
     "hideDust": {
       "title": "Ocultar pagamentos pequenos",
       "description": "Ocultar pagamentos menores que 0,1 XLM"
+    },
```

Comment:
looks like we are missing some pt translations? E.g.: `addMemoExplanationBottomSheet`

Reply:
and `transactionSettingsBottomSheet`